packages/shared/src/components/feeds/FeedContainer.tsx

```diff
@@ -100,6 +106,11 @@ export const FeedContainer = ({
   shortcuts,
   actionButtons,
 }: FeedContainerProps): ReactElement => {
+  const { value: isShortcutsV1 } = useConditionalFeature({
+    feature: feature.onboardingMostVisited,
+    shouldEvaluate: !!shortcuts,
+    returnDefault: true,
```

Comment:
Not sure yet why we needed, but would suggest a naming of `allowReturnDefault` rather then the current one since it can be mistaken as if the func will always return default

Reply:
It won't always return default though.
In the default way it won't even get evaluated, thus return undefined forever.